src/app/components/mno-local-products/mno-local-product.coffee

```diff
@@ -42,6 +52,13 @@ angular.module 'mnoEnterpriseAngular'
           productId = $stateParams.productId
           vm.product = _.findWhere(vm.products, { nid: productId })
           vm.product ||= _.findWhere(vm.products, { id: productId })
+          # Is currency selection enabled
+          currencySelection = MnoeConfig.isCurrencySelectionEnabled()
+          # Are there any available plans
+          availablePlans = ProvisioningHelper.plansForCurrency(vm.product.pricing_plans, vm.orgCurrency)
+
+          vm.orderPossible = !_.isEmpty(availablePlans) || (vm.product.pricing_plans&[0].default&[0] && currencySelection)
```

Comment:
Safe navigation in coffeescript is `?` 

vm.product.pricing_plans?[0].default?[0